metamorphic/build.go

```diff
@@ -170,9 +208,11 @@ func buildForIngestExternalEmulation(
 	defer reader.Close()
 
 	writable := objstorageprovider.NewFileWritable(f)
-	meta, err := writeSSTForIngestion(
+	meta, _, err := writeSSTForIngestion(
 		t,
 		pointIter, rangeDelIter, rangeKeyIter,
+		false, /* uniquePrefixes */
```

Comment:
hrm, maybe i guess it doesn't matter because we create the actual external sst without a prefix h[ere](https://github.com/RaduBerinde/pebble/blob/meta-suffix-replacement/metamorphic/ops.go#L1783)